Engine/engine.py

```diff
@@ -33,6 +33,8 @@ class Engine(Process):
     DEFAULT_CAMERA_NUMBER = 4
     DEFAULT_FPS_LOCK_STATE = True
     DEFAULT_FPS = 30
+    PROFILE_DATA_TIME = 10
```

Comment:
I know it's not related to that PR, but could you add a comment in from of DEFAULT_FPS to remember to use optionnal argument?
```python
DEFAULT_FPS = 30 # Please don't change this constant, instead run the AI with the optionnal argument --engine_fps 
```